docs/docs/how-tos/branching.ipynb

```diff
@@ -341,31 +356,48 @@
     "\n",
     "from typing_extensions import TypedDict\n",
     "\n",
-    "from langgraph.graph import END, START, StateGraph\n",
+    "from langgraph.graph import StateGraph, START, END\n",
```

Comment:
we need to remove or rewrite the line above

```
"If you have a known \"sink\" node that the conditional branches will route to afterwards, you can provide `then=<final-node-name>` when creating the conditional edges."
```

we are not actually using `then` here